.github/insist.sh

```diff
@@ -0,0 +1,11 @@
+insist () {
```

Comment:
if you insist on this name :-D
what can I say ;-)

I'd call it max attempt test for RC 0 or something like that, but it's just nitpicking